thoth/adviser/run.py

```diff
@@ -132,7 +132,7 @@ def subprocess_run(
         _, exit_code = os.waitpid(pid, 0)
         if exit_code != 0:
             _LOGGER.error("Child exited with exit code %r", exit_code)
-            if exit_code == 9 or exit_code == 137:
```

Comment:
what do these exit codes mean?

Reply:
The child process used for resolving software stacks signalizes to the parent what went wrong. It is documented in sources:

https://github.com/thoth-station/adviser/blob/1112a52bff469c610b3b3a428904e837ff0d89bc/thoth/adviser/run.py#L124-L127